compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp

```diff
@@ -98,20 +98,18 @@ bool check_4d_transpose(loco::Node *node, const std::vector<int32_t> indices)
   return true;
 }
 
-luci::CircleTranspose *create_4d_transpose(luci::CircleNode *node,
+luci::CircleTranspose *create_Nd_transpose(luci::CircleNode *node,
```

Comment:
This change has internal relation but not direct. I recommand to separate this change.
And it would be better to leave `create_4d_transpose` as is and add new `create_Nd_transpose` .

Reply:
I separated out `create_Nd_transpose`, but, if possible, can I keep it in this PR? This way, we introduce a helper method and use it in the same pull request.